### DISCUSSION TITLE
添加第三方库的头文件会找不到头文件，但是编译可以正常编译。

### DISCUSSION THREAD

**Discussion by zhangxichang** [xmake-io/xmake#5502] [Q&A]
添加第三方库的头文件会找不到头文件，但是编译可以正常编译。我执行VScode的更新intellisense命令也仍然提示找不到头文件。
我在Windows平台使用msvc编译的，这是我的项目文件内容：
```
add_rules("mode.debug", "mode.release")

add_requires("godotcpp4")

target("godot-spine")
    set_kind("shared")
    add_files("./src/*.cpp")
    add_packages("godotcpp4")
```

**Comment by star-hengxing** [comment]
看一下
https://xmake.io/#/plugin/more_plugins?id=configure-intellsence

**Reply by zhangxichang** [reply]
我按照文档改用clangd问题解决了，但是不知道为什么使用vscode-cpptools不能智能提示

**Reply by star-hengxing** [reply]
cpptools 也可以设置 compile_commands.json
https://code.visualstudio.com/docs/cpp/c-cpp-properties-schema-reference

**Reply by zhangxichang** [reply]
谢谢，解决了
